<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathReformat</class>
 <widget class="qMRMLWidget" name="PathReformat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>404</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector">
        <property name="toolTip">
         <string>Select a curve markup or a VMTK centerline model</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsCurveNode</string>
          <string>vtkMRMLMarkupsClosedCurveNode</string>
          <string>vtkMRMLModelNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Position:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ctkSliderWidget" name="positionIndexSliderWidget">
        <property name="toolTip">
         <string>Set position along path</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>500.000000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
        <property name="spinBoxVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>2D view</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="viewGroupBox">
        <property name="toolTip">
         <string>Select a 2D view</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="viewGroupBoxLayout">
         <item>
          <widget class="QRadioButton" name="redRadioButton">
           <property name="text">
            <string>Red</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="greenRadioButton">
           <property name="text">
            <string>Green</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="yellowRadioButton">
           <property name="text">
            <string>Yellow</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="qMRMLCollapsibleButton" name="moreCollapsibleButton">
     <property name="text">
      <string>More</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Hide path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="hideCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLCollapsibleButton</class>
   <extends>ctkCollapsibleButton</extends>
   <header>qMRMLCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>PathReformat</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
